tests/test_bijectors.py

```diff
@@ -92,3 +93,116 @@ def test_ramp_bijector_inverse_exp():
 
 
 
+def _make_inv_forward_MixtureAffineSigmoidBijector(t, a, b, c, nb_dimension, nb_component):
```

Comment:
hummmm I don;t think it is necessary here to create a separate function. You are using a different function for each test, so it might as well be directly the test function itself. Otherwise this is a bit cluttered.

Here is how I would write this function:
```python
def test_bijectorMixtureAffineSigmoidBijector_inverse():
  """
  Testing that the inverse transform of MixtureAffineSigmoidBijector is indeed the inverse of 
  forward.
  """
  batch_size = 100

  for params in _test_params_MASB:
    x = jax.random.uniform(jax.random.PRNGKey(0), shape=[batch_size,params['nb_dimension']])
    
    # shape for components of the transform
    shape = [batch_size, params['nb_dimension'], params['nb_component']]
    bij = MixtureAffineSigmoidBijector(
                              jnp.ones(shape)*a,
                              jnp.ones(shape)*b,
                              jnp.ones(shape)*c,
                              jax.nn.softmax(jax.random.uniform(jax.random.PRNGKey(1), shape=shape)))

    assert_allclose(bij.inverse(bij.forward(x)*1.), x, rtol=2e-2, atol=1e-5)
```
It's just one function instead of 2, a lot more compact :-)